### DISCUSSION TITLE
How to track how users navigate the theme?

### DISCUSSION THREAD

**Discussion by rachel-rigdon-sp** [squidfunk/mkdocs-material#4237] [Q&A] [answered]
We want to understand how our users navigate our docs so we can see if they're using search or the nav options. 

Has anyone used a heat map type of product on their docs? Or found other ways to track how users are navigating the theme? We ran one on our docs and were surprised to find that large product icons and the bottom nav were primarily used. And dismayed to find that search isn't being used much.

**Comment by squidfunk** [comment, marked as answer]
We're using Google Analytics ([natively integrated](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#google-analytics)) to learn about documentation usage. I think the newer version of GA can also print heatmaps. In any case, you can integrate any JavaScript-based tracking solution via [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript).

**Reply by rachel-rigdon-sp** [reply]
We're using Google Analytics (UA) for data as well, although we're still learning how to really interpret and use it (but that's a different conversation). I'll look into those two options. Thank you for your help!
